src/common/config_opts.h

```diff
@@ -881,6 +881,9 @@ OPTION(osd_deep_scrub_interval, OPT_FLOAT, 60*60*24*7) // once a week
 OPTION(osd_deep_scrub_randomize_ratio, OPT_FLOAT, 0.15) // scrubs will randomly become deep scrubs at this rate (0.15 -> 15% of scrubs are deep)
 OPTION(osd_deep_scrub_stride, OPT_INT, 524288)
 OPTION(osd_deep_scrub_update_digest_min_age, OPT_INT, 2*60*60)   // objects must be this old (seconds) before we update the whole-object digest on scrub
+OPTION(osd_deep_scrub_large_omap_object_key_threshold, OPT_U64, 2000000)   // threshold for number of keys to determine a large omap object
```

Comment:
Where'd 200k come from? I think that's larger than CephFS will get these days, and is certainly larger than RBD ever got. It may still be too small for RGW though?